fe/fe-core/src/main/java/com/starrocks/qe/SetExecutor.java

```diff
@@ -54,13 +54,25 @@ private void setVariable(SetVar var) throws DdlException {
             // do nothing
             return;
         } else {
-            VariableMgr.setVar(ctx.getSessionVariable(), var);
+            VariableMgr.setVar(ctx.getSessionVariable(), var, false);
         }
     }
 
-    public void execute() throws DdlException {
+    private boolean isSessionVar(SetVar var) {
+        return !(var instanceof SetPassVar
+                || var instanceof SetNamesVar
+                || var instanceof SetTransaction);
+    }
+
+    public void execute(boolean onlySessionVar) throws DdlException {
```

Comment:
What's does the onlySessionVar meaning?

Reply:
If onlySessionVar is true, it will only set the session variables, and ignore others variables.